### DISCUSSION TITLE
Grey promo card variation

### DISCUSSION THREAD

**Discussion by github-tma8118** [Legal-and-General/canopy#1138] [Canopy requests]
**Type of request**
Modification

**Component, pattern, template, helper or foundation?**
Component

**Nature of request**

For our app, we want to include a few promo cards that look like this:

![promo card](https://user-images.githubusercontent.com/109232127/214842150-31095200-e608-44e4-a00b-29f5d3160a5a.png)

It looks to me like it could be a new grey variant of the existing promo cards, but without the image, or maybe a variant of a card component.

Could you please advise on what you think would be best here?  🙇‍♂️

**Comment by paul-request** [comment]
Technically this is possible. We would have to introduce a new variable to be able to make the card header a different color to the default variant font color. My only concern is taht if we made some sort of `dark` variant, would this work with other components that use variants already? @alexcanning

**Reply by paul-request** [reply]
Looks like a variant of a card component to me

**Reply by elenagarrone** [reply]
@paul-request I was thinking this could be added as a new variant of the `lgVariant` directive, what do you think? 

https://legal-and-general.github.io/canopy/?path=/docs/helpers-directives-variant-guide--page

**Reply by paul-request** [reply]
Yeah, although my only concern was the header colour and whether there would be other child elements that would need to be changed to a different colour as the variant directive only changes the `background` and the `color` properties

**Comment by alexcanning** [comment]
Left some commentary for James so expect we might see some updated designs

**Comment by alexcanning** [comment]
Closing as this will be taken care of as part of the work to revamp the promo cards.
